docs/source/index.rst

```diff
@@ -0,0 +1,81 @@
+Documentation
+=============
+
+Executor
+---------
+The primary object in pywren is an `executor`. The standard way to get everything set up is to import pywren, and call the `default_executor` function.
+
+.. autofunction:: pywren.wren.default_executor
+
+`default_executor()` reads your `pywren_config` and returns `executor` object that's ready to go.
+We can use this `executor` to run `map`, which applies a function to a list of data in the cloud.
+
+.. automethod:: pywren.executor.Executor.map
+
+`map` returns a list of `futures`, and which can return their `result` when the task is complete. 
+
+.. autoclass:: pywren.future.ResponseFuture
+
+    .. automethod:: pywren.future.ResponseFuture.result
+
+Waiting for the results
+--------------------------
+
+.. autofunction:: pywren.wait.wait
+
+Alternatively, if you want to wait for everything to finish and then get all of the results, you can simply call `get_all_results`
+
+.. autofunction:: pywren.wren.get_all_results
+
+
+Standalone Mode
+---------------
+
+To run pywren in standalone mode, run
+
+.. code-block:: bash
+
+  pywren standalone launch_instances 50
+
+This launches EC2 instances to run pywren. Once the instances are ready, you can run pywren as usual. 
```

Comment:
Can we add a comment on what instance types are used etc ? (or a link to some other page which explains it)